apps/meteor/client/views/omnichannel/triggers/TriggersTableContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import { Callout } from '@rocket.chat/fuselage';
+import { useEndpoint, useTranslation } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+import React, { useState } from 'react';
+
+import type { GenericTableParams } from '../../../components/GenericTable/GenericTable';
+import TriggersTable from './TriggersTable';
+
+const TriggersTableContainer = () => {
+	const t = useTranslation();
+	const [params, setParams] = useState<GenericTableParams>(() => ({ current: 0, itemsPerPage: 25 }));
+
+	const { current, itemsPerPage } = params;
+
+	const getTriggers = useEndpoint('GET', '/v1/livechat/triggers');
+	const { data, refetch, isError } = useQuery(['/v1/livechat/triggers'], () => getTriggers({ offset: current, count: itemsPerPage }));
+
+	if (isError) {
+		return <Callout>{t('Error')}: error</Callout>;
+	}
+
+	return (
+		<TriggersTable
+			triggers={data?.triggers || []}
+			totalTriggers={data?.triggers.length || 0}
```

Comment:
```suggestion
			totalTriggers={data?.total || 0}
```
This one has the correct total items. Also, you will need to update the `'/v1/livechat/triggers'` endpoint type return to add the `total: number` there.